OMMatlab.m

```diff
@@ -696,18 +695,18 @@ function simulate(obj,resultfile,simflags)
                 buildmodelexpr=join(["buildModel(",cNames(1),")"]);
                 buildModelmsg=obj.sendExpression(buildmodelexpr);
                 %disp(buildModelmsg(:))
-                if(~isempty(buildModelmsg(1)))
+                if(~(buildModelmsg(1)=="")) %This is a changed code

```

Comment:
It does not. However, the buildModelmsg(1) is never empty in any iteration. Even if the buildModelmsg(1) is an empty string, it still does not return false. It is because it is not empty, but it is just a zero character string with the result "".  Therefore, this if conditional would always be true. Using isempty on strings is therefore not recommended since it is mostly applicable to arrays with numbers.

Reply:
This is is fixed with this commit https://github.com/OpenModelica/OMMatlab/commit/4a68ace515be1c41b19b08713f0809dd42c275b3, the `isempty` built in function works on char arrays